aries-site/src/data/wcag/rangeinput.json

```diff
@@ -0,0 +1,512 @@
+  },
+  {
+    "rule": "1.3.2",
+    "label": "Meaningful Sequence",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.3",
+    "label": "Sensory Characteristics",
+    "status": "conditional",
+    "notes": "Must provide clear instructions for assissitive technologies."
+  },
+  {
+    "rule": "1.3.4",
+    "label": "Orientation",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.5",
+    "label": "Identify Input Purpose",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.6",
+    "label": "Identify Purpose",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.1",
+    "label": "Use of Color",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.2",
+    "label": "Audio Control",
+    "status": "not-applicable",
+    "notes": "Assuming that the range input isn't being used to control audio in any use cases?"
+  },
+  {
+    "rule": "1.4.3",
+    "label": "Contrast (Minimum)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.4",
+    "label": "Resize Text",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.5",
+    "label": "Images of Text",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.6",
+    "label": "Contrast (Enhanced)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.7",
+    "label": "Low or No Background Audio",
+    "status": "not-applicable",
+    "notes": "Assuming that the range input isn't being used to control audio in any use cases?"
+  },
+  {
+    "rule": "1.4.8",
+    "label": "Visual Presentation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.9",
+    "label": "Images of Text (No Exception)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.10",
+    "label": "Reflow",
+    "status": "conditional",
+    "notes": "Should not implemented to require horizontal scrolling. Must be fully visible on smaller screens and zoom levels."
+  },
+  {
+    "rule": "1.4.11",
+    "label": "Non-text Contrast",
+    "status": "failed",
+    "notes": "- Filled portion of the track vs non filled portion fails contrast ratio requirements when range input is placed on top of background default, front, and back.\n- Filled portion of the track fails contrast ratio requirements when range input is placed on top of background back.\n- (Dark mode) Thumb vs non filled portion of the track fails contrast ratio requirements when range input is placed on top of background default, back, and front\n- (Dark mode) Focus indicator vs non filled portion of the track fails contrast ratio when range input is placed on top of background default, back, and front\n(Dark mode) Focus indicator fails contrast ratio when range input is placed on top of background default, back, and front"
+  },
+  {
+    "rule": "1.4.12",
+    "label": "Text Spacing",
+    "status": "passed",
```

Comment:
I wonder since the actual RangeInput component itself doesnt have Text if this would be not-applicable

Reply:
@britt6612 are the min and max values technically not a part of the rangeinput?

Reply:
https://design-system.hpe.design/components/rangeinput?q=range

Reply:
yeah technically in that example their just added `text` on the sides of the RangeInput our actual RangeInput doesnt support the idea of a "label" on the sides

Reply:
Gotcha okay, not-applicable makes sense to me then.